lego/tests/test_lego.py

```diff
@@ -15,42 +15,42 @@ class TestsSpecA:
     @classmethod
     @pytest.mark.lego('giraffe')
     def setup_class(cls, components):
```

Comment:
rename to connections. fix everywhere and rerun tests to verify.

Reply:
I ran my tests..
Will run your also

Reply:
Done